woodpecker-go/woodpecker/client.go

```diff
@@ -187,109 +186,109 @@ func (c *client) RepoMove(owner, name, newFullName string) error {
-	uri := fmt.Sprintf(pathBuild, c.addr, owner, name, num)
+	uri := fmt.Sprintf(pathPipeline, c.addr, owner, name, num)
 	err := c.post(uri+"?"+val.Encode(), nil, out)
 	return out, err
 }
 
-// BuildStop cancels the running job.
-func (c *client) BuildStop(owner, name string, num, job int) error {
+// PipelineStop cancels the running job.
+func (c *client) PipelineStop(owner, name string, num, job int) error {
 	uri := fmt.Sprintf(pathJob, c.addr, owner, name, num, job)
 	err := c.delete(uri)
 	return err
 }
 
-// BuildApprove approves a blocked build.
-func (c *client) BuildApprove(owner, name string, num int) (*Build, error) {
-	out := new(Build)
+// PipelineApprove approves a blocked pipeline.
+func (c *client) PipelineApprove(owner, name string, num int) (*Pipeline, error) {
+	out := new(Pipeline)
 	uri := fmt.Sprintf(pathApprove, c.addr, owner, name, num)
 	err := c.post(uri, nil, out)
 	return out, err
 }
 
-// BuildDecline declines a blocked build.
-func (c *client) BuildDecline(owner, name string, num int) (*Build, error) {
-	out := new(Build)
+// PipelineDecline declines a blocked pipeline.
+func (c *client) PipelineDecline(owner, name string, num int) (*Pipeline, error) {
+	out := new(Pipeline)
 	uri := fmt.Sprintf(pathDecline, c.addr, owner, name, num)
 	err := c.post(uri, nil, out)
 	return out, err
 }
 
-// BuildKill force kills the running build.
-func (c *client) BuildKill(owner, name string, num int) error {
-	uri := fmt.Sprintf(pathBuild, c.addr, owner, name, num)
+// PipelineKill force kills the running pipeline.
+func (c *client) PipelineKill(owner, name string, num int) error {
+	uri := fmt.Sprintf(pathPipeline, c.addr, owner, name, num)
 	err := c.delete(uri)
 	return err
 }
 
-// BuildLogs returns the build logs for the specified job.
-func (c *client) BuildLogs(owner, name string, num, job int) ([]*Logs, error) {
+// PipelineLogs returns the pipeline logs for the specified job.
+func (c *client) PipelineLogs(owner, name string, num, job int) ([]*Logs, error) {
 	uri := fmt.Sprintf(pathLogs, c.addr, owner, name, num, job)
 	var out []*Logs
 	err := c.get(uri, &out)
 	return out, err
 }
 
-// Deploy triggers a deployment for an existing build using the
+// Deploy triggers a deployment for an existing pipeline using the
 // specified target environment.
-func (c *client) Deploy(owner, name string, num int, env string, params map[string]string) (*Build, error) {
-	out := new(Build)
+func (c *client) Deploy(owner, name string, num int, env string, params map[string]string) (*Pipeline, error) {
+	out := new(Pipeline)
 	val := mapValues(params)
 	val.Set("event", "deployment")
 	val.Set("deploy_to", env)
-	uri := fmt.Sprintf(pathBuild, c.addr, owner, name, num)
+	uri := fmt.Sprintf(pathPipeline, c.addr, owner, name, num)
 	err := c.post(uri+"?"+val.Encode(), nil, out)
 	return out, err
 }
 
-// LogsPurge purges the build logs for the specified build.
+// LogsPurge purges the pipeline logs for the specified pipeline.
```

Comment:
NOTE to me: oh we already have it in backend ... we should expose log deletion for webUI too (isn't there an issue already? CB ?!?)

Reply:
https://github.com/woodpecker-ci/woodpecker/issues/1100

Reply:
no that's about a pipeline not about it's logs

Reply:
Thanks